agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -378,10 +368,6 @@ private void validateTestwiseCoverageConfig(Validator validator) {
 				httpServerPort != null && testImpactConfig.testEnvironmentVariable != null,
 				"You did set both 'http-server-port' and 'test-env'! Only one of them is allowed!");
 
-		validator.isFalse(uploadUrl != null,
-				"'upload-url' option is " +
-						"incompatible with Testwise coverage mode!");
-
```

Comment:
So after reading this part, I was wondering if we need `additionalMetaDataFiles` at all anymore, but I realized that they are used in the contexdt of artifactory and azure uploads. However, this line suggests that they need to be used in combination with the `uploadUrl` which I found strange. Can you explain how this can work?

Reply:
I think this comment is probably set on the wrong change (at least for me it is) ^^' 
I replied here: https://github.com/cqse/teamscale-jacoco-agent/pull/401/files#r1459138353